server/index.html

```diff
@@ -37,4 +55,4 @@
   </div>
   <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
   </body>
-</html>
\ No newline at end of file
+</html>
```

Comment:
Ah, the neverending fight over whether plain text files should have a newline at the end. Stand strong comrade.